### PR TITLE
I1355

### PR DESCRIPTION
**Pull request by hillalex**
Scripts to set up both machines, and run a one-off backup of configured directories

### CODE REVIEW

File: setup_starport.sh [hunk 1]
```diff
@@ -18,6 +11,12 @@ vault write secret/annex/id_rsa value=@$KEY_PATH/id_rsa
...
 cat $KEY_PATH/id_rsa.pub >> ~/.ssh/authorized_keys
-rm -r $KEY_PATH
 
-echo "Setup of Starport complete. You can now set up production"
\ No newline at end of file
+function cleanup {
+    set +e
+    rm -r $KEY_PATH
+}
+trap cleanup EXIT
```
Comment (MartinEden) [inline setup_starport.sh:14]: I think we want to setup this trap just after the `mkdir` line - at the end of the script is does nothing much.

**Review by MartinEden — CHANGES_REQUESTED** [review]
Nice :) Just one remaining issue from my point of view, but I agree that @richfitz should review the SSH side of things.

File: setup_starport.sh (hunk 1 above)
Comment (hillalex) [reply setup_starport.sh:14]: oops, good spot. moved.

**Comment by MartinEden** [comment]
Alex, one of the changes hasn't made it to this PR I think: Splitting up the config keys so that the starport config is a dictionary.

**Comment by hillalex** [comment]
That one was committed to master!

**Pull request merged**